examples/distributed-collections/itemlisteners/ItemListener.cpp

```diff
@@ -60,7 +60,8 @@ int main() {
     std::cout << "Registered the listener with registration id:" << registrationId <<
     "Waiting for the listener events!" << std::endl;
 
-    hazelcast::util::sleep(5);
+    //Sleep until the execution of CollectionChanger is finished.
```

Comment:
We probably need to put a README in the test folder to tell the user which one to start first and do what, step by step if we are having these kinds of requests. Just assume that the user knows nothing. Furthermore, we need to make it as esy as possible for the user to try an example, therefore let's put a runExample.sh and runExample.bat scripts to run the script and let the script do the sleep, what do you think?